utils.py

```diff
@@ -364,7 +354,7 @@ def getHTMLLogString(base_path: str = "./buildbot") -> str:
 <html>
 <body>' >> {base_path}/mysql_logs.html
 
-echo '<a href=" {os.getenv('ARTIFACTS_URL', default='https://ci.mariadb.org')}/%(prop:tarbuildnum)s/logs/%(prop:buildername)s/">logs (mariadbd.gz + var.tar.gz)</a><br>' >> {base_path}/mysql_logs.html
+echo '<a href=" {os.environ['ARTIFACTS_URL']}/%(prop:tarbuildnum)s/logs/%(prop:buildername)s/">logs (mariadbd.gz + var.tar.gz)</a><br>' >> {base_path}/mysql_logs.html
```

Comment:
Nitpick: There's an extra space here that's probably not intentional `href=" {...` instead of `href="{...`